p2p/net/swarm/dial_worker_test.go

```diff
@@ -342,3 +358,206 @@ func TestDialWorkerLoopConcurrentFailureStress(t *testing.T) {
 	close(reqch)
 	worker.wg.Wait()
 }
+
+func TestDialWorkerLoopRanking(t *testing.T) {
+	s1 := makeSwarm(t)
+	s2 := makeSwarm(t)
+	defer s1.Close()
+	defer s2.Close()
+
+	var quicAddr, tcpAddr ma.Multiaddr
+	for _, a := range s2.ListenAddresses() {
+		if _, err := a.ValueForProtocol(ma.P_QUIC); err == nil {
+			quicAddr = a
+		}
+		if _, err := a.ValueForProtocol(ma.P_TCP); err == nil {
+			tcpAddr = a
+		}
+	}
+
+	tcpL1, silAddr1 := makeTcpListener(t)
```

Comment:
what is `sil` short for? maybe expand it to something else?

Reply:
removed this test. hopefully the new one is clearer. 
sil was silent. It hangs up after accepting.